templates/vue/cypress/integration/import-export.spec.js

```diff
@@ -1,28 +1,187 @@
+        typeData: {
+          ...node.typeData,
+          activity: {
+            questions: [
+              {
+                text: "Question",
+                followUp: {
+                  enabled: false,
+                  text: "",
+                  nodeId: null,
+                  questionId: null,
+                },
+                answerTypes: {
+                  text: {
+                    enabled: false,
+                    placeholder: "",
+                    isMultiLine: false,
+                    allowMultiple: false,
+                    minFields: 1,
+                    maxFields: 100,
+                  },
+                  audio: { enabled: false },
+                  dragDrop: {
+                    enabled: true,
+                    buckets: [
+                      {
+                        id: "3d37b350-0cc2-4ec9-a2e5-ad90cd37fef0",
+                        type: "from",
+                        text: "A",
+                      },
+                      {
+                        id: "f732da5c-3356-4869-a634-f186228656c9",
+                        type: "to",
+                        text: "B",
+                      },
+                    ],
+                    items: [
+                      {
+                        id: "fc0bb16a-f38b-4961-8bd2-c7d58a2f1555",
+                        color: "#808080",
+                        text: "Item",
+                        imageUrl: nonexistentImageUrl,
+                        bucketId: "3d37b350-0cc2-4ec9-a2e5-ad90cd37fef0",
+                      },
+                    ],
+                    useImages: true,
+                  },
+                  multipleChoice: {
+                    enabled: false,
+                    allowSelectMultiple: false,
+                    useImages: false,
+                    choices: [],
+                    preSelectedOptions: [],
+                  },
+                },
+                confirmation: { title: "", message: "" },
+                completed: false,
+                optional: false,
+                id: "5a457076-8065-4bd4-8bf0-fdf525b62937",
+              },
+            ],
+          },
+        },
+      })
+    })
+
+    cy.get(".settings-button").click()
+    cy.contains(/advanced/i).click()
+    cy.getByTestId("export-type-toggle").click()
+    cy.contains(/Export as ZIP/i)
+      .should("be.visible")
+      .click()
+
+    cy.wait("@export_zip")
+      .its("response.body")
+      .then(data => {
+        expect("zipUrl" in data).to.be.true
+        expect("wpPosts" in data).to.be.false
+      })
+
+    cy.contains(".alert-warning", /exported with warnings/i).should("be.visible")
+    cy.getByTestId("export-warnings-table")
+      .should("be.visible")
+      .get("tr")
+      .find("li")
+      .should("have.length", 2)
+  })
+})
+
+describe("Import", () => {
+  beforeEach(() => {
     cy.setup()
+
     cy.intercept("PUT", "**/tapestries/**").as("import")
```

Comment:
I think importing a Tapestry makes sense as `PUT /tapestries/:tapestryPostId`, since it replace an existing Tapestry with the contents of the imported Tapestry.

Using the `POST` method was the best way I could find to receive uploaded files through PHP. This is why the `import_zip` endpoint uses `POST`, but ideally, I think it should have been PUT as well.